tests/clients/js/wrapper.go

```diff
@@ -0,0 +1,442 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+//go:build js
+
+package js
```

Comment:
question: Does this execute the js client as wasm in a JS runtime?

Reply:
Yes, everything (including the test framework) is executed in a NodeJS runtime.

Reply:
Nice! That's really great, thanks Keenan.